lib/drivers/onetouch/oneTouchUltra2.js

```diff
@@ -213,43 +199,73 @@ module.exports = function (config) {
      }
 
     // get data from lines
-    var dpattern = /P "(\w+)","(\d{2}\/\d{2}\/\d{2})","(\d{2}:\d{2}:\d{2})   ","..(\d{3}).","(.)","(\d{2})",\W00\W(.+)/;
+    var dpattern = /P "(\w+)","(\d{2}\/\d{2}\/\d{2})","(\d{2}:\d{2}:\d{2})   ","(..\d{3}).","(.)","(\d{2})",\W00\W(.+)/;
```

Comment:
True, it's currently ignoring the parity error flag. I think the correct response would be to drop the reading if there is a parity error. I'll update this right now.